src/pd/client.rs

```diff
@@ -108,6 +108,18 @@ pub trait PdClient: Send + Sync + 'static {
         })
         .boxed()
     }
+
+    // Wrap ranges in a Stream.
+    fn stores_for_ranges(
+        self: Arc<Self>,
+        ranges: Vec<BoundRange>,
+    ) -> BoxStream<'static, Result<(Vec<BoundRange>, Store<Self::KvClient>)>> {
+        self.region_for_key(&Key::from(Vec::new()))
```

Comment:
What is happening here? It seems a bit odd to find the region for the empty key, rather than something in `ranges`.

Reply:
Yeah, I do think it's odd. I just want to get store here.

Reply:
I don't think it's correct here. You should find the stores and regions by the ranges.

Reply:
Oh I understand, if the ranges are in several stores, than I should make multiple requests right?

Reply:
Yes. Every request can only match one region.

Reply:
Thanks, I'll update it!